### Manta

#### SV calling

```
python configManta.py --bam pto1.bam --bam pto2.bam --bam pto3.bam ... --runDir ../manta_result/ --referenceFasta genome.fasta 
```

#### Quality control
```
awk '/^#/ || ($7 == "PASS" && $6 > 20 && $8 !~ /IMPRECISE/)' pto1.vcf > pto1_precise.vcf
```

####  Merge all individual VCF result files

```
svimmer --threads 60 vcf.list Chr01 Chr02 Chr03 Chr04 <......> # chromosome
```

#### Population Genotypingï¼Œperformed on different chromosomes

```bash
graphtyper genotype_sv genome.fasta sample_merged_sort.vcf.gz --sams=bam.list --region Chr01 --threads 40
```

#### Merge

```bash
echo Chr{01..19} ChrX | tr ' ' '\n' | while read chrom; do if [[ ! -d sv_results/${chrom} ]]; then continue; fi; find sv_results/${chrom} -name "*.vcf.gz" | sort; done > vcf_file_list
bcftools concat --naive --file-list vcf_file_list -Oz -o graphtyper.vcf.gz
awk '/^#/ || ($5 ~ /AGGREGATED|INV/)' graphtyper.vcf > vcf_filter.vcf
awk '/^#/ || ($7 == "PASS")' vcf_filter.vcf > vcf_allPASS.vcf

```
